Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -32,7 +32,7 @@ variable {α : Type*}
 
 /-- A linearly ordered commutative monoid with a zero element. -/
 class LinearOrderedCommMonoidWithZero (α : Type*) extends LinearOrderedCommMonoid α,
-  CommMonoidWithZero α where
+  CommMonoidWithZero α, OrderBot α where
```

Comment:
So now we get `0 = \bot` as a lemma. But `zero_eq_bot` and `zero_eq_bot'` don't apply, I think.
What is the recommended way to do a rewrite now?